docs/telemetry.md

```diff
@@ -0,0 +1,42 @@
+# Telemetry
+
+The Telemetry function enables the Operator gathering and sending basic anonymous data to Percona, which helps us to determine where to focus the development and what is the uptake for each release of Operator. 
+
+The following information is gathered:
+
+* ID of the Custom Resource (the `metadata.uid` field)
+* Kubernetes version
+* Platform (is it Kubernetes or Openshift)
+* PMM Version
+* Operator version
+* PostgreSQL version
+* PgBackRest version
+
+We do not gather anything that identify a system, but the following thing should be mentioned:
+Custom Resource ID is a unique ID generated by Kubernetes for each Custom Resource.
+
+Telemetry is enabled by default and is sent to the [Version Service server](update.md#automatic-upgrade) when the Operator connects to it at scheduled times to obtain fresh information about version numbers and valid image paths needed for the upgrade.
+
+The landing page for this service, [check.percona.com](https://check.percona.com/), explains what this service is.
+
+You can use [your own Version Service](update.md#automatic-upgrade) to avoid
+telemetry data leave the space of your cluster, or you can disable sending
```

Comment:
this is not true. even if you use your own version service but telemetry is not disabled, we'll still send data to `check.percona.com`. only way to disable telemetry is setting `disable_telemetry: true`